central/development/service/service_impl_test.go

```diff
@@ -136,3 +139,127 @@ func (s *developmentServiceAccessControlTestSuite) TestDevelopmentServiceRandomB
 		})
 	}
 }
+
+func (s *developmentServiceAccessControlTestSuite) TestCertService() {
+	cases := []struct {
+		url               string
+		certPEM           string
+		expectedResp      central.URLHasValidCertResponse_URLResult
+		additionalMessage string
+	}{
+		{
+			url: "https://untrusted-root.badssl.com",
+			certPEM: `
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you extend the validity of the certificates to unlimited? They are expiring in 2026, I've run into this issue in the `TLSChallengeTest` multiple times and regretted it.

Reply:
50 years should be fine